qcelemental/physical_constants/context.py

```diff
@@ -2,12 +2,15 @@
 Contains relevant physical constants
 """
 
-import collections
-import pint
 from decimal import Decimal
+import collections
 
-from .. import datum
+try:
```

Comment:
Maybe put this in either the `p/c` init or the main `init` to make sure this is the first instance imported.

Reply:
sure. added analogous pydantic warning to #14, too.